priv/app/graph_panel.jsx

```diff
@@ -62,16 +62,13 @@ export default class GraphPanel extends React.Component {
   render() {
     var funs = this.state.funs;
 
-    var graphsPanels = [];
-    for (var i = 0; i < funs.length; i++) {
-      graphsPanels.push(
-        <div key={funs[i]} className="row">
-          <div className="col-md-12">
-            <Graph removeGraph={this.removeGraph.bind(this)} mfa={funs[i]}/>
-          </div>
+    var graphsPanels = funs.reverse().map(fun =>
+      <div key={fun} className="row">
```

Comment:
Are `fun` is good key here? fun is a reference to array which is "rereferenced" around each 500ms (method `handleFuns()`). I think `key={fun[3]}` will be better here. Key should be a unique identifier for this div, so that react is able to perform better DOM diff.

Reply:
Yeah, good point. But about this PR, it should be closed as @gomoripeti introduced changes to backend that enabled showing graphs in reverse order. I will close this PR in a minute.

Reply:
Article from react documentation - https://reactjs.org/docs/reconciliation.html#recursing-on-children

Reply:
just for the record there is `utils.chartId(fun)` to create a safe id from 4th element of fun. As far as I can see it is not used right now. (Was used with our previous graph lib `flot`)